ci/unit/test_main_node.sh

```diff
@@ -44,15 +44,18 @@ else
 fi
 
 cd "$DAOS_BASE"
-IS_CI=true OLD_CI=false RUN_TEST_VALGRIND="$WITH_VALGRIND" utils/run_test.sh
-mkdir -p vm_test
+if [ -z "$NLT" ]; then
```

Comment:
Since `$NLT` is boolean:

```suggestion
if ${NLT:-false}; then
```